_posts/en/posts/2018-06-07-release-0.16.1.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Bitcoin Core 0.16.1 Released
+name: blog-release-0.16.1
+id: en-blog-release-0.16.1
+lang: en
+type: posts
+layout: post
+share: true
+version: 1
+
+excerpt: >
+  Bitcoin Core 0.16.1 has been released with the latest bug fixes and
+  minor updates
+---
+Bitcoin Core version 0.16.1 is now available for [download][download
+page].  All users are encouraged to upgrade to this [maintenance
+release][] that fixes several bugs and provides backports of new minor
+features, such as:
+
+- Mitigating a vector for denial-of-service attacks.  This attack required
+  compromising particular services and would've probably been most
+  effective against nodes that were started for the first time (rather
+  than nodes that had already been connected to the network for more
+  than a few hours).
+
+- Fixing a bug that could've potentially caused miners to lose revenue if
+  they produced two blocks in very rapid succession.
+
+- Removing a previously-deprecated option related to maximum block size
+  in favor of an option that sets maximum block weight.  This change
+  only affects miners.  The interaction between these two options
```

Comment:
`The interaction between these two options sometimes confused miners, and this change simplifies the situation.` isn't isn't quite right. `-blockmaxsize` has been silently ignored since 0.15.1. This change simply makes it explicit that `-blockmaxsize` doesn't do anything.